### PR TITLE
[eclipse/xtext#1814] Use 'basic' pod template for simrel job

### PR DESCRIPTION
**Pull request by kthoms**
I have configured the job https://ci.eclipse.org/xtext/job/releng/job/release-simrel-update-issue1814/ to use the branch.

It is failing ATM, but later and for different reason. I assume the basic Git problem is solved with the change.

### CODE REVIEW

File: releng/jenkins/release-simrel-update/Jenkinsfile [hunk 1]
```diff
@@ -80,7 +76,10 @@ pipeline {
                 XTEXT_VERSION_LABEL=$(echo $XTEXT_VERSION | sed "s|\\(.*\\)\\.v.*|\\1.$RELEASE_TYPE|");;
             esac
             git diff
-            git commit --signoff -m "Xtext $XTEXT_VERSION_LABEL contribution to SimRel $SIMREL_LABEL"
+            git --version
+            git commit --signoff -m "Xtext $XTEXT_VERSION_LABEL contribution to SimRel $SIMREL_LABEL
+            
+Change-Id: I0000000000000000000000000000000000000000"
```
Comment (cdietrich) [inline releng/jenkins/release-simrel-update/Jenkinsfile:82]: is this really the problem?

Comment (kthoms) [reply releng/jenkins/release-simrel-update/Jenkinsfile:82]: No, this slipped in accidentially. Only agent and grep command should change. Sorry for that, will revert these lines now.

Comment (kthoms) [reply releng/jenkins/release-simrel-update/Jenkinsfile]: @cdietrich updated

**Comment by cdietrich** [comment]
do you have an idea why the test job is red?

**Comment by kthoms** [comment]
The job was started with specifying XTEXT_VERSION=2.23.0. This is not correct, the parameter must contain the qualified version. The qualified version is normally passed in by the upstream build. Here I entered that manually.
I'm sure the build would succeed when the qualified version is passed in.

**Comment by cdietrich** [comment]
so we dont know if this will work until we do the next milestone?

**Comment by Hejado** [comment]
> 
> 
> so we dont know if this will work until we do the next milestone?

unless we want to stage and then withdraw a fake milestone/release, yes... I'm okay with that though...

**Pull request merged**